dnainator-javafx/src/main/java/nl/tudelft/dnainator/javafx/drawables/strains/Strain.java

```diff
@@ -182,4 +188,26 @@ private void loadRank(Integer key, List<Cluster> value) {
 			childContent.getChildren().add(cluster);
 		}
 	}
-}
+
+	/**
+	 * @return The minimum rank property.
+	 */
+	public IntegerProperty minRankProperty() {
+		return minRank;
+	}
+
+	/**
+	 * @return The maximum rank property.
+	 */
+	public IntegerProperty maxRankProperty() {
+		return maxRank;
+	}
+
+	/**
+	 * Get the rank width.
+	 * @return the rank width.
+	 */
+	public int getRankWidth() {
```

Comment:
This function really shouldn't exist xD.
I know I've used it somewhere as an example, but the rankwidth is not a property we should expose.